Project-I-Social-Card/social-card/src/components/HeaderComponents/Header.css

```
@@ -1 +1,45 @@
 /* All styles for your header Components live here */
+
+.header{
+    
+    width:100%;
+    
+    display:flex;
+    flex-wrap:wrap;
+    flex-direction: row;
+}
+
+.title{
+    
+   
+    width:60%;
```

```suggestion
    width:90%;
```
I think the title area takes up more space than that, try 90% instead.